src/pages/iou/request/step/IOURequestStepScan/index.tsx

```diff
@@ -327,17 +311,19 @@ function IOURequestStepScan({
                 {!isDraggingOver && (Browser.isMobile() ? mobileCameraView() : desktopUploadView())}
                 <ReceiptDropUI
                     onDrop={(e) => {
-                        const file = lodashGet(e, ['dataTransfer', 'files', 0]);
-                        setReceiptAndNavigate(file);
+                        const file = e?.dataTransfer?.files[0];
+                        if (file) {
+                            setReceiptAndNavigate(file);
+                        }
                     }}
                     receiptImageTopPosition={receiptImageTopPosition}
                 />
                 <ConfirmModal
-                    title={attachmentInvalidReasonTitle ? translate(attachmentInvalidReasonTitle) : ''}
+                    title={attachmentInvalidReasonTitle ? translate(attachmentInvalidReasonTitle as TranslationPaths) : ''}
```

Comment:
you can type `attachmentInvalidReasonTitle` as `TranslationPaths` so then assertion wont be needed here